UITests/localhostLoad.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html>
+<head>
+<script>
+  var json = '{"history":["javascript:alert(\'Local page loaded\')"],"currentPage":-1}';
+  var sessionURL = 'http://localhost:6571/about/sessionrestore?history=' + escape(json);
+  var errorURL = 'http://localhost:6571/errors/error.html?url=nttps://mozilla.com/';
+
+  function newTabExploit() {
+    var win = window.open("http://1.2.3.4.5");
```

Comment:
Is this intentionally a bad URL? I think this will get parsed as a bad IP address?